massa-models/src/mapping_grpc.rs

```diff
@@ -98,12 +102,14 @@ impl From<Endorsement> for grpc_model::Endorsement {
 
 impl From<SecureShareEndorsement> for grpc_model::SignedEndorsement {
     fn from(value: SecureShareEndorsement) -> Self {
+        let serialized_size = value.serialized_size() as u64;
```

Comment:
you may want to inline the variable it will be more consistent with other field of the struct

Reply:
good catch... this kind of stuff  should detected by the linter